SPECS/qtbase/qtbase.spec

```diff
@@ -92,67 +89,28 @@ Patch8: tell-the-truth-about-private-api.patch
 
 # upstreamable patches
 # namespace QT_VERSION_CHECK to workaround major/minor being pre-defined (#1396755)
-Patch50: qtbase-opensource-src-5.8.0-QT_VERSION_CHECK.patch
-
-# 1381828 - Broken window scaling for some QT5 applications (#1381828)
-# This patch moves the threshold for 2x scaling from the DPI of 144 to 192,
-# the same value GNOME uses. It's not a complete solution...
-Patch51: qtbase-hidpi_scale_at_192.patch
+Patch50: qtbase-QT_VERSION_CHECK.patch
 
 # 1. Workaround moc/multilib issues
 # https://bugzilla.redhat.com/show_bug.cgi?id=1290020
 # https://bugreports.qt.io/browse/QTBUG-49972
 # 2. Workaround sysmacros.h (pre)defining major/minor a breaking stuff
-Patch52: qtbase-opensource-src-5.7.1-moc_macros.patch
-
-# CMake generates wrong -isystem /usr/include compilations flags with Qt5::Gui
-# https://bugzilla.redhat.com/1704474
-Patch53: qtbase-everywhere-src-5.12.1-qt5gui_cmake_isystem_includes.patch
-
-# respect QMAKE_LFLAGS_RELEASE when building qmake
-Patch54: qtbase-qmake_LFLAGS.patch
+Patch52: qtbase-moc-macros.patch
```

Comment:
multilib (32 and 64 bit support on the same system) is not supported in Azure Linux.  You may be able to remove this.

Reply:
patch removed, does not impact the build.